docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -78,12 +78,29 @@ Configure the `azure-cli` repository information as shown previously. Available
     apt-cache policy azure-cli
     ```
 
-2. To install specific version:
+1. To install specific version with variables:
 
     ```bash
-    sudo apt-get install azure-cli=<version>-1~bullseye
+    # Obtain the currently installed distribution
+    AZ_DIST=$(lsb_release -cs)
+
+    # Store an Azure CLI version of choice
+    AZ_VER=2.51.0
+
+    # Install a specific version
+    sudo apt-get install azure-cli=$AZ_VER-1~$AZ_DIST
     ```
 
+1. To install a specific version without variables, replace the Azure CLI version and Linux distribution name shown:
+
+    ```bash
+    sudo apt-get install azure-cli=2.51.0-1~bullseye
+    ```
+
+If you receive error message _Unable to locate package azure-cli_, the `azure-cli` repository information is not configured.  Follow the steps given in [Option 2: Step-by-step installation instructions](#option-2-step-by-step-installation-instructions).
+
+If you receive error message _Version x.xx.x-1~distributionName for azure-cli was not found_, use `lsb_release -cs` to find the distribution you're running.
+
 ## Update Azure CLI
```

Comment:
But the step `2` and `3` are confusing, it looks like that both steps are required.
I suggest merging them to step 2 and tell users to choose any of them explicitly.
![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/2227874/2fcc361e-08a0-443d-a1ad-21db157a0480)

Reply:
@bebound, good point.